cmd/makeConfig.go

```diff
@@ -59,7 +59,6 @@ func emptyConfig() []byte {
 		Name:   "default",
 		UserID: "",
 		Token:  "",
-		URL:    defaultAPIURL,
```

Comment:
Why is this gone?

Reply:
Forgot to reimplement this, had to remove because i used a different url (to test my build with dev faster)

Reply:
And because there's no longer need of client.go, but as mentioned its reimplemented now